ui/packages/catalog-ui-search/src/main/webapp/extension-points/extension-points.tsx

```diff
@@ -33,6 +34,7 @@ export type ExtensionPointsType = {
   visualizations: any[]
   queryForms: any[]
   navigationRight: any[]
+  metacardInteractions: any[]
```

Comment:
Is there a way to type these in a more helpful way, such as React.ReactNode?

Reply:
I looked at a downstream project and they use things that are not React Components (ie. a local variable)

Reply:
Talked @adimka, he's going to make downstream changes and I'll change this to be of type ReactNode